### DISCUSSION TITLE
Welcome to top-rss-list Discussions!

### DISCUSSION THREAD

**Discussion by weekend-project-space** [weekend-project-space/top-rss-list#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by xiu19** [comment]
建议
增加批量取消订阅rss源，
适当限制订阅数量
手机端网页浏览文章中部切换下个文章，不能自动回到顶部

祝你越来越好

**Reply by weekend-project-space** [reply]
后的，会改进的

**Comment by TANG42** [comment]


**Reply by weekend-project-space** [reply]
可以，进入订阅源信息列表后左上角有取消订阅按钮

**Comment by You-Hdd** [comment]
这是我今天网上无意间发现这个网站。
我再使用的过程中，发现这个网站获取某些RSS源最新咨询的时候。
有些RSS源获取不到，但是我手机上也有个叫Read you APP。两个产品用RSS源都一样的。手机可以都获取，但是网站有的却获取不到。不知道是什么情况，是bug还是什么。这个read you APP在GitHub上也有该项目。
作者如果与兴趣的话，能否跟read you的作者合作一下，提供账户功能，使其可以在手机端和PC端能同步就好

**Reply by weekend-project-space** [reply]
获取不到的源的地址方便提供一下吗？

**Reply by You-Hdd** [reply]
[常用 RSS 源列表](https://[plink.anyfeeder.com](https://plink.anyfeeder.com/)/)
我是导入这个网址里的 OPML 文件的RSS源的
麻烦你测试下

**Reply by You-Hdd** [reply]
[常用RSS源列表](https://plink.anyfeeder.com/)

**Reply by weekend-project-space** [reply]
已调整，数据抓取有时会超时导致
